web/src/lib/domain-logic/businessPersonaHelpers.test.ts

```diff
@@ -0,0 +1,216 @@
+      expect(
+        isStartingBusiness(
+          generateBusiness({
+            profileData: generateProfileData({
+              businessPersona: "OWNING",
+            }),
+          })
+        )
+      ).toBe(false);
+    });
+
+    it("returns true when business persona is starting", () => {
+      expect(
+        isStartingBusiness(
+          generateBusiness({
+            profileData: generateProfileData({
+              businessPersona: "STARTING",
+            }),
+          })
+        )
+      ).toBe(true);
+    });
+  });
+
+  describe("isNexusBusiness", () => {
+    describe("foreign business persona", () => {
+      const Business = (override: ForeignBusinessTypeId[]): Business => {
+        return generateBusiness({
+          profileData: generateProfileData({ foreignBusinessTypeIds: override, businessPersona: "FOREIGN" }),
+        });
+      };
+
+      it("returns true when employeeOrContractorInNJ", () => {
+        expect(isNexusBusiness(Business(["employeeOrContractorInNJ"]))).toBe(true);
+      });
+
+      it("returns true when officeInNJ", () => {
+        expect(isNexusBusiness(Business(["officeInNJ"]))).toBe(true);
+      });
+
+      it("returns true when propertyInNJ", () => {
+        expect(isNexusBusiness(Business(["propertyInNJ"]))).toBe(true);
+      });
+
+      it("returns true when companyOperatedVehiclesInNJ", () => {
+        expect(isNexusBusiness(Business(["companyOperatedVehiclesInNJ"]))).toBe(true);
+      });
+
+      it("returns false when empty", () => {
+        expect(isNexusBusiness(Business([]))).toBe(false);
+      });
+
+      it("returns false when employeesInNJ", () => {
+        expect(isNexusBusiness(Business(["employeesInNJ"]))).toBe(false);
+      });
+
+      it("returns false when revenueInNJ", () => {
+        expect(isNexusBusiness(Business(["revenueInNJ"]))).toBe(false);
+      });
+
+      it("returns false when transactionsInNJ", () => {
+        expect(isNexusBusiness(Business(["transactionsInNJ"]))).toBe(false);
+      });
+
+      it("returns false when business is undefined", () => {
+        expect(isNexusBusiness(undefined)).toBe(false);
+      });
+    });
+
+    it("returns false when business persona is not Foreign", () => {
+      expect(
+        isNexusBusiness(
+          generateBusiness({
+            profileData: generateProfileData({
+              foreignBusinessTypeIds: ["employeesInNJ"],
+              businessPersona: "STARTING",
+            }),
+          })
+        )
+      ).toBe(false);
+    });
+  });
+
+  describe("isRemoteWorkerOrSellerBusiness", () => {
+    describe("foreign business persona", () => {
+      const Business = (override: ForeignBusinessTypeId[]): Business => {
+        return generateBusiness({
+          profileData: generateProfileData({ foreignBusinessTypeIds: override, businessPersona: "FOREIGN" }),
+        });
+      };
+
+      it("returns true when employeesInNJ", () => {
```

Comment:
[pebble] Same comment as above. We can reduce the number of tests here and make this more flexible.